processor/src/host/dsa.rs

```diff
@@ -1,18 +1,23 @@
 use alloc::vec::Vec;
 
+#[cfg(feature = "std")]
+use vm_core::crypto::hash::Rpo256;
 use vm_core::{Felt, Word};
 
 use crate::ExecutionError;
 
 /// Gets as input a vector containing a secret key, and a word representing a message and outputs a
 /// vector of values to be pushed onto the advice stack.
+///
 /// The values are the ones required for a Falcon signature verification inside the VM and they are:
 ///
-/// 1. The nonce represented as 8 field elements.
+/// 1. The challenge point at which we evaluate the three aforementioned polynomials to check the
+///    product relationship.
```

Comment:
I'm not sure the word "aforementioned" is correct here. Also, I would specify that the challenge point is 2 field elements.

Reply:
Indeed, this was from the previous order of the bullet points. Changed it now and updated the comment.